entries/QUnit.config.xml

```diff
@@ -50,7 +55,7 @@
 		</property>
 		<property name="testId" type="Array" default="undefined">
 			<desc>
-				This property allows QUnit to run specific test blocks by a hashed string combining their module name and test name. You can specify one or multiple tests to run.
+				This property allows QUnit to run specific tests identified by the hashed version of their module name and test name. You can specify one or multiple, comma-separated tests to run.
```

Comment:
> Normally I'd disagree with this, but given that I just learned you can do `?moduleId=foo&moduleId=bar` (meaning comma-separated is _not_ the only way to specify multiple values) I'm good with the original wording.

Correct, and note also that this is documentation for `QUnit.config` as opposed to QUnit URL parameters (which [doesn't yet exist](https://github.com/jquery/api.qunitjs.com/issues/117)).